<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnplay1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>460</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnplay2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>460</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QSlider" name="hzslidervs">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>490</y>
      <width>160</width>
      <height>29</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblVoz">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>490</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Velocidad de voz</string>
    </property>
   </widget>
   <widget class="QSlider" name="hzslidervt">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>520</y>
      <width>160</width>
      <height>29</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblVoz_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>520</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tono de voz</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxLang">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>560</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblVoz_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>560</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Acento</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblrules">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Reglas</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>110</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>190</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnaplicar">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>230</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Aplicar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblreemplazar">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>330</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Reemplazar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit1">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>360</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>360</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblcon">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>370</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>por</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtload">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>271</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtres">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>40</y>
      <width>271</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnreplace">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>400</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Reemplazar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblVoz_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>600</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo de voz</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxVoice">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>600</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>886</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionCargar_Archivo"/>
    <addaction name="actionGuardar_Audio_Generado"/>
    <addaction name="actionGuardar_Texto_Generado"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuAcciones">
    <property name="title">
     <string>Acciones</string>
    </property>
    <addaction name="actionAbrir_ventana"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuAcciones"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCargar_Archivo">
   <property name="text">
    <string>Cargar Archivo</string>
   </property>
  </action>
  <action name="actionGuardar_Audio_Generado">
   <property name="text">
    <string>Guardar Audio Generado</string>
   </property>
  </action>
  <action name="actionGuardar_Texto_Generado">
   <property name="text">
    <string>Guardar Texto Generado</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionAbrir_ventana">
   <property name="text">
    <string>Abrir nueva ventana</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
